### ISSUE TITLE
Starting animation from AnimationEnd

### ISSUE THREAD

**Issue by sorenbs** [craftyjs/Crafty#228]
It should be possible to start an animation from a function bound to AnimationEnd

```
    player = Crafty.e("2D, Canvas, animator, SpriteAnimation")
        .attr({x: 64, y: 64, z: 2})
        .animate("level1", 0, 0, 5)
        .animate("level2", 0, 1, 5)
        .animate("level1", 30, 2)
    player.bind("AnimationEnd", 
        function()
        {
            this.animate("level1",30, 5);
            console.log("AnimationEnd");
        });
```

http://jsfiddle.net/KD8q9/24/

**Comment by ahilles107** [comment]
this still don't work. I works on this but without results. But i fixed #275. Will be cool if anyone can look at this.

**Comment by BlackScorp** [comment]
i think we need AnimationEnd and AnimationStop events AnimationEnd will be triggered after each Reel animation Stop after animation ends

**Comment by ahilles107** [comment]
sounds good for me!

**Comment by starwed** [comment]
Should be fixed by #426
